routes.txt

```diff
@@ -0,0 +1,6 @@
+route | NUS | 
+node | BUS | 1123 | NUS | start here | 0.0 | 0.0
```

Comment:
Is this file going to be committed?

Reply:
No, I am still cleaning up the code. This has been resolved already.

Reply:
One more routes.txt below